go.mod

```
@@ -30,74 +22,53 @@ require (
-	github.com/hashicorp/go-sockaddr v1.0.2 // indirect
 	github.com/hashicorp/go-version v1.2.1
-	github.com/hashicorp/golang-lru v0.5.4 // indirect
-	github.com/hashicorp/memberlist v0.2.4 // indirect
 	github.com/influxdata/influxdb1-client v0.0.0-20200827194710-b269163b24ab
 	github.com/jjcollinge/servicefabric v0.0.2-0.20180125130438-8eebe170fa1b
 	github.com/libkermit/compose v0.0.0-20171122111507-c04e39c026ad
 	github.com/libkermit/docker v0.0.0-20171122101128-e6674d32b807
 	github.com/libkermit/docker-check v0.0.0-20171122104347-1113af38e591
-	github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369 // indirect
 	github.com/mesos/mesos-go v0.0.3-0.20150930144802-068d5470506e
 	github.com/mesosphere/mesos-dns v0.0.0-00010101000000-000000000000
 	github.com/miekg/dns v1.1.43
 	github.com/mitchellh/copystructure v1.0.0
-	github.com/mitchellh/go-testing-interface v1.14.1 // indirect
 	github.com/mitchellh/hashstructure v1.0.0
 	github.com/mitchellh/mapstructure v1.4.1
-	github.com/mitchellh/reflectwalk v1.0.1 // indirect
 	github.com/mvdan/xurls v1.1.1-0.20170309204242-db96455566f0
 	github.com/ogier/pflag v0.0.2-0.20160129220114-45c278ab3607
-	github.com/opencontainers/image-spec v1.0.0-rc5.0.20170515205857-f03dbe35d449 // indirect
-	github.com/opencontainers/runc v1.0.0-rc3.0.20170425215914-b6b70e534517 // indirect
-	github.com/opentracing-contrib/go-observer v0.0.0-20170622124052-a52f23424492 // indirect
-	github.com/opentracing/opentracing-go v1.0.2
+	github.com/opentracing/opentracing-go v1.1.0
 	github.com/openzipkin-contrib/zipkin-go-opentracing v0.3.5
 	github.com/patrickmn/go-cache v2.1.0+incompatible
-	github.com/philhofer/fwd v1.0.0 // indirect
 	github.com/prometheus/client_golang v1.11.0
-	github.com/prometheus/client_model v0.2.0
+	github.com/prometheus/client_model v0.4.0
 	github.com/rancher/go-rancher v0.1.1-0.20171004213057-52e2f4895340
 	github.com/rancher/go-rancher-metadata v0.0.0-00010101000000-000000000000
 	github.com/ryanuber/go-glob v1.0.0
 	github.com/shopspring/decimal v1.1.1-0.20191009025716-f1972eb1d1f5
 	github.com/sirupsen/logrus v1.8.1
-	github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify v1.8.3
 	github.com/stvp/go-udp-testing v0.0.0-20171104055251-c4434f09ec13
 	github.com/thoas/stats v0.0.0-20190104110215-4975baf6a358
-	github.com/tinylib/msgp v1.0.2 // indirect
-	github.com/tv42/zbase32 v0.0.0-20150911225513-03389da7e0bf // indirect
 	github.com/uber/jaeger-client-go v2.15.0+incompatible
 	github.com/uber/jaeger-lib v1.5.0
 	github.com/unrolled/render v0.0.0-20170109143244-50716a0a8537
 	github.com/unrolled/secure v1.0.5
 	github.com/urfave/negroni v0.2.1-0.20170426175938-490e6a555d47
 	github.com/vdemeester/shakers v0.1.0
 	github.com/vulcand/oxy v1.2.0
-	go.etcd.io/bbolt v1.3.5 // indirect
-	golang.org/x/net v0.0.0-20211209124913-491a49abca63
-	google.golang.org/grpc v1.38.0
+	golang.org/x/net v0.17.0
+	google.golang.org/grpc v1.52.0-dev
```

`v1.52.0-dev` doesn't address the issue.
```suggestion
	google.golang.org/grpc v1.58.3
```

---

Ah good catch thanks.

What happened was when I ran `go get github.com/containous/go-bindata/...` locally, it would modify go.mod and downgrade grpc version. I've changed the .drone.yml to do a `go install` instead which doesn't modify go.mod.